code/mc_1_20_6/src/main/java/org/betonquest/betonquest/mc_1_20_6/BundledMC_1_20_6.java

```diff
@@ -12,35 +13,40 @@
  * Allows to register features with Minecraft 1.20.6.
  */
 @SuppressWarnings("PMD.ClassNamingConventions")
-public class BundledMC_1_20_6 {
+public class BundledMC_1_20_6 implements Integrator {
 
     /**
-     * Custom Logger instance for this class.
+     * BetonQuest class to get relevant object from.
      */
-    private final BetonQuestLogger log;
+    private final BetonQuest betonQuest;
 
     /**
      * Creates a new Object to register Minecraft version changes.
      *
-     * @param log the custom logger for this class
+     * @param betonQuest the BetonQuest class to get relevant object from
      */
-    public BundledMC_1_20_6(final BetonQuestLogger log) {
-        this.log = log;
+    public BundledMC_1_20_6(final BetonQuest betonQuest) {
```

Comment:
Wouldn't it be more advisable to replace it with proper dependency injection instead of using the implementation class BetonQuest?